src/gbt.cpp

```diff
@@ -60,23 +44,32 @@ bool scanGBTPhasesLocal(localArgs *la, const uint32_t ohN, const uint32_t N, con
     for (uint8_t phase = phaseMin; phase <= phaseMax; phase += phaseStep) {
         // Set the new phases
         for (uint32_t vfatN = 0; vfatN < oh::VFATS_PER_OH; vfatN++) {
-            if (writeGBTPhaseLocal(la, ohN, vfatN, phase))
-                return true;
+            writeGBTPhase{}(ohN, vfatN, phase);
         }
 
         // Wait for the phases to be set
         std::this_thread::sleep_for(std::chrono::milliseconds(10));
 
-        for (uint32_t repN = 0; repN < N; repN++) {
+        std::stringstream linkGood_regName, syncErrCnt_regName, cfgRun_regName;
```

Comment:
I glossed over the finer points of your argument (logging vs register names) and was focusing on the "do we need to put a numerical type in this or not" question...
If it's cleaner code, then I'm all for just creating registers with plain `std::string` and `std::to_string` as needed (my tests show that there is minimal performance difference)

Reply:
it ends up being quite a bit longer since you have to use ``string::append`` commands anyways. Here's a snippet of the register naming with ``std::string`` implementation:

```
                 linkGood_regName = "GEM_AMC.OH_LINKS.OH";
                 linkGood_regName.append(std::to_string(ohN));
                 linkGood_regName.append(".VFAT");
                 linkGood_regName.append(std::to_string(vfatN);
                 linkGood_regName.append(".LINK_GOOD");
                 syncErrCnt_regName = "GEM_AMC.OH_LINKS.OH";
                 syncErrCnt_regName.append(std::to_string(ohN));
                 syncErrCnt_regName.append(".VFAT");
                 syncErrCnt_regName.append(std::to_string(vfatN);
                 syncErrCnt_regName.append(".SYNC_ERR_CNT");
                 cfgRun_regName = "GEM_AMC.OH.OH";
                 cfgRun_regName.append(std::to_string(ohN));
                 cfgRun_regName.append(".GEB.VFAT");
                 cfgRun_regName.append(std::to_string(vfatN);
                 cfgRun_regName.append(".CFG_RUN");
```

Reply:
This can be shortened using `operator+`:
```C++
linkGood_regName = "GEM_AMC.OH_LINKS.OH" + std::to_string(ohN) + ".VFAT" + std::to_string(vfatN) + ".LINK_GOOD";
```
etc